concat.hpp

```diff
@@ -0,0 +1,119 @@
+#include "utils.hpp"
```

Comment:
Please, add a copyright and basic descriptive header.

Reply:
Done